### DISCUSSION TITLE
Taskitem Padding/Margin and Font Customizations

### DISCUSSION THREAD

**Discussion by mattparkes** [lawand-dot-io/taskbar#87] [Feature requests]
I've just changed from uBar to taskbar and whilst 50% of this is just getting used to the change, I do think being able to add some (customisable) horizontal and vertical padding to each taskbar item would be a big help, along with changing the font, etc.

**Comment by lawand-dot-io** [comment]
@mattparkes Hello. Thanks for the request. This is a duplicate of https://github.com/lawand-dot-io/taskbar/discussions/36

So, don't worry, this is planned to be added before Taskbar starts requiring purchase :) I hope you continue to find Taskbar useful in the meantime. Cheers
